backend/api/swagger/experiment.swagger.json

```diff
@@ -186,7 +188,7 @@
     },
     "/apis/v1beta1/experiments/{id}:archive": {
       "post": {
-        "summary": "Archive an experiment.",
+        "summary": "Archive an experiment. Runs and jobs inside the experiment will be archived as well.",
```

Comment:
Suggested revision:

Archives an experiment and the experiment's runs and jobs.

Reply:
addressed in the proto file